src/lib/tracker.ts

```diff
@@ -277,21 +277,29 @@ export default function Plausible(
     }
   ) => {
     function trackClick(this: HTMLAnchorElement, event: MouseEvent) {
-      if (event.ctrlKey) return; // Don't send event on Ctrl + Click, should behave like middle-mouse-click.
+      // auxclick is fired for any mouse-button except primary (left).
+      // The event should only be sent on aux (middle) click, return for others (right, etc).
+      const auxEvent = event.type === 'auxclick';
+      if (auxEvent && event.button !== 1) return;
 
       trackEvent('Outbound Link: Click', { props: { url: this.href } });
 
+      // _self (same as falsy), _parent and _top need a delay for the request to go through.
+      // _blank opens a new tab -> no need for delay
+      // no special handling for custom targets -> use the default browser behavior
       /* istanbul ignore next */
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
       if (
+        (!this.target || this.target.match(/^_(self|parent|top)$/i)) &&
+        !auxEvent && // middle mouse click opens a new tab -> no need for delay
+        !event.ctrlKey && // ctrl + click opens a new tab -> no need for delay
+        !event.metaKey && // same as ctrl
+        !event.shiftKey && // shift + click opens a new window -> no need for delay
+        // don't try to access location object during testing
         !(
           typeof process !== 'undefined' &&
           process &&
           process.env.NODE_ENV === 'test'
```

Comment:
What are these for exactly? I assume it's to avoid errors during testing but there are already quite a bunch of errors when running `yarn test` regarding non-implemented navigation features and I can only assume it's because we're clicking links and that tries to run code to navigate to it's href but I've not found a way to confirm this.

Reply:
I'm not 100% sure since I didn't write it but I believe the idea is to not call `setTimeout` in tests. Maybe @Maronato has more information for this